packages/strapi-parts/src/Button/Button.js

```diff
@@ -45,9 +58,15 @@ export const Button = React.forwardRef(
 
     return (
       <ButtonWrapper ref={ref} aria-disabled={disabled} size={size} variant={variant} onClick={handleClick} {...props}>
-        {startIcon && (
+        {(startIcon || loading) && (
```

Comment:
Shouldn't it be disabled during the loading state?

Reply:
I agree, I think @maevalienard will be helpful since on the design, the loading state is shown on a primary variant (which is not the disabled one)

Reply:
(I've pushed the disabled thing 😊 )